\documentclass[9pt,t]{beamer}
\usetheme{PastelGG}
\usepackage{tikz}
\usepackage{amsthm,amsmath,amsfonts,mathbbol,mathrsfs,stmaryrd,textcomp,hyperref}

%%-----------------------------------------------------------------------
%% Add a progress bar in footline
%% http://www.mrunix.de/forums/showpost.php?p=316577&postcount=3
%% ----------------------------------------------------------------------
\definecolor{lightgr}{rgb}{0.8235294117647058 0.615686274509804 0.6980392156862745}
\makeatletter
\addtobeamertemplate{footline}{%
  \color{lightgr}% to color the progressbar
  \hspace*{-\beamer@leftmargin}%
  \rule{\beamer@leftmargin}{2pt}%
  \rlap{\rule{\dimexpr
      \beamer@startpageofframe\dimexpr
      \beamer@rightmargin+\textwidth\relax/\beamer@endpageofdocument}{1pt}}
  % next 'empty' line is mandatory!

  \vspace{0\baselineskip}
  {}
}

\title{FRY: A Fast Approximation to ROAST Gene Set Test with Mean Aggregated Set Statistics}
\author[G\"{o}knur Giner]{G\"{o}knur Giner}
\date[November 1st, 2016]{November 1st, 2016,\\
AB$^3$ACBS Conference, Brisbane, AU}


\begin{document}

\begin{frame}[plain,t]
	\titlepage
\end{frame}

\section{Introduction}
\begin{frame}[c]
	\frametitle{Differential Expression Analysis}
	\vspace{0.2cm}
\begin{columns}
	\column{0.5\textwidth}
	\textbf{\color{oxygenpurple} Extract mRNA and turn into cDNA}
\begin{minipage}[c][0.35\textheight][c]{\linewidth}
  \includegraphics[width=1\linewidth]{figures/de1_no_title}
\end{minipage}
 	\visible<2->{\textbf{\color{oxygenpurple} Sequence the fragments}
\begin{minipage}[c][0.25\textheight][c]{\linewidth}
  \centering
  \includegraphics[width=0.7\linewidth]{figures/de2_no_title}
\end{minipage}}
	\visible<3->{
	\textbf{\color{oxygenpurple} Align reads to the reference genome}
\begin{minipage}[c][0.3\textheight][c]{\linewidth}
  \includegraphics[width=1\linewidth]{figures/de3_no_title}
\end{minipage}}
	\column{0.5\textwidth}
 	\visible<4->{
	\textbf{\color{oxygenpurple} Gene counts and count matrix}
\begin{minipage}[c][0.25\textheight][c]{\linewidth}
 	\centering
  \includegraphics[width=1.1\linewidth]{figures/de4_no_title}
\end{minipage}}
 	\visible<5->{
	\textbf{\color{oxygenpurple} Differential expression analysis}
\begin{minipage}[c][0.90\textwidth][c]{\linewidth}
  	\centering
  \includegraphics[width=0.95\linewidth]{figures/de5_no_title}
\end{minipage}}
\end{columns}
\end{frame}

% \begin{minipage}[pos][height][contentpos]{width} text \end{minipage} 
% pos: c(center), t(top), b(bottom) control the vertical alignment
% contentpos: c(center), t(top), b(bottom), s(spread)
% width: width of the box
\begin{frame}
	\frametitle{A Collection of Genes}
\begin{columns}
	\column{0.6\textwidth}
\begin{minipage}[c][1\textheight][c]{1\columnwidth}
 	\centering
  \includegraphics[width=1\columnwidth]{figures/bcl2}
\end{minipage}
	\column{0.4\textwidth}
	\visible<2->{
\begin{minipage}[c][1\textheight][c]{0.9\columnwidth}
\textbf{\color{oxygenpurple} Annotation Databases}
 	\centering
  \includegraphics[width=1\columnwidth]{figures/AnnotationDB}
\end{minipage}}
\end{columns}
\end{frame}

\begin{frame}[c]
\frametitle{Existing Methods}
	 \centering
	 \vfill
	 \includegraphics[width=1\textwidth]{figures/tools}
\end{frame}

%\begin{frame}
%	\frametitle{Gene Set Tests}
%	\vspace{2cm}
%	{\color{oxygenpurple}\textbf{Biologists want to:}}\\
%	\vspace{0.4cm}
%\begin{itemize}
%	\item Assemble a summary of the most enriched biology
%	\item Understand if the biological pathway of interest is changed in a given experiment
%\end{itemize}
%\end{frame}

\begin{frame}
\frametitle{Null Hypotheses in Gene Set Tests}
\begin{itemize}
	\item Self-Contained
	\item Competitive
\end{itemize}
\end{frame}

\begin{frame}
	\frametitle{Input Sources for Gene Set Tests}
	\vspace{0.1cm}
	\begin{minipage}[t]{0.3\textwidth}
	\centering
	{\color{oxygenpurple}\textbf{DNA Microarray}}
	\includegraphics[width=1\textwidth]{figures/DNA_Microarray}
	\end{minipage}
	\hspace{0.4cm}
	\begin{minipage}[t]{0.3\textwidth}
	\centering
	{\color{oxygenpurple}\textbf{RNA Sequencing}}
	\vspace{-0.5cm}
	\includegraphics[width=1\textwidth]{figures/RNA_Sequencing}
	\end{minipage}
	\hspace{0.2cm}
	\begin{minipage}[t]{0.3\textwidth}
	\centering
	{\color{oxygenpurple}\textbf{Pathway Databases}}
	\vspace{-0.5cm}
	\includegraphics[width=0.6\textwidth]{figures/databases}
	\end{minipage}
\end{frame}

\begin{frame}
\frametitle{Test Methods}
		\begin{itemize}
		\item Over Representation
		\item Functional Class Scoring
		\end{itemize}
\end{frame}


\begin{frame}
	\frametitle{Assessment of Statistical Significance}
	\vspace{0.1cm}
	\begin{columns}
	\column{0.5\textwidth}	
	\centering
    \textbf{\color{oxygenpurple} Permutation Tests}
	\vbox to 0.5\textheight
	{
	\vspace{0.02\textwidth}
	%\vspace{0.1cm}
	\centering
    \includegraphics[width=0.7\textwidth]{figures/permutation}                
	\vspace{0.02cm}
	\begin{itemize}
		\item Require large number of samples 
		\item[]
		\item Applicable to limited type of experimental design
		\item[]
		\item GSEA is based on permutation test
		\end{itemize}
    }                       
	\column{0.5\textwidth}
	\centering
	{\color{oxygenpurple}\textbf{Rotation Tests}}
	\begin{itemize}
	\item Can produce unlimited number of rotated test statistics
	\item[]
	\item Applicable to complex arbitrary experimental designs
	\item[]	
	\item Roast: Rotation gene set tests for complex microarray experiments
	\end{itemize}
	\vspace{0.02cm}
	\includegraphics[width=0.6\textwidth]{figures/rotation}
	\end{columns}
\end{frame}

\begin{frame}
	\frametitle{Exploring Biological Processes in Breast Cancer}
	\centering
	\includegraphics[width=0.8\textwidth]{figures/breast_cancer_experiment}
\end{frame}

\begin{frame}
\frametitle{Barcode Plot}
\begin{minipage}[t][][b]{0.45\textwidth}	
	\begin{itemize}
		\item Both bi-directional
		\uncover<2-5>{\item[]
		\item Both applicable to arbitrary designs
		\item[]}
		\uncover<3-5>{
		\item Both work well with small sample sizes
		\item[]}
		\uncover<4-5>{
		\item Roast uses floor mean, mean50, mean square and mean to aggregate gene set level statistics
		\item[]}
		\uncover<5>{
		\item Fry uses only mean to aggregate gene level statistics}
	\end{itemize}
\end{minipage}
\begin{minipage}[t][][b]{0.54\textwidth}
	\includegraphics[width=1.1\textwidth]{figures/barcode_plot}
\end{minipage}
\end{frame}

\begin{frame}
\frametitle{Downsides of Roast}
Limitations when analysing large collections of sets\\
\vfill
 \uncover<2-4>{\textbf{\color{oxygenpurple} P-value resolution is limited by the number of rotations}}\\
\vfill
\uncover<3-4>{Top detected sets are all have the same p-values}\\
\vfill
\uncover<4>{P-values for each set may vary from run to run}\\
\vfill
\visible<2-4>{\begin{minipage}{1.4\textheight}
\includegraphics[width=1.1\textwidth]{figures/roast_table_short2}
\end{minipage}}
\end{frame}

\begin{frame}
\frametitle{Improved Features with Fry}
Assumes equal gene-wise variances\\
\vfill
 \uncover<2-5>{Gene level statistics are standardized using posterior gene variances} \\
\vfill
  \uncover<3-5>{\textbf{\color{oxygenpurple} Produces high-resolution exact p-values unlike roast}}\\
\vfill
\uncover<4-5>{Top detected sets are all have different p-values}\\
\vfill
\uncover<5>{P-values for each set does not vary from run to run}\\
\vfill
\visible<3-5>{\begin{minipage}{1.4\textheight}
\includegraphics[width=1.1\textwidth]{figures/fry_table_short2}
\end{minipage}}
\end{frame}

\begin{frame}
\frametitle{Fry is computationally very efficient}
\vfill
\large\textbf{\color{oxygenpurple}Over 4000 pathways are tested on an experiment with $15000+$ genes and $800+$ samples of TCGA Breast carcinoma simultaneously! Roast is rotated $10e5$ times!}\\
\vfill
\includegraphics[width=1\textwidth]{figures/speed}
\end{frame}

\begin{frame}
\frametitle{Fry outperforms Roast}
\begin{figure}
\includegraphics[width=1.2\textheight]{figures/power1}
\end{figure}
\end{frame}
\begin{frame}
\frametitle{Impact of Outdated Gene Annotations}
\vspace{0.2cm}
A very recent and comprehensive study on analysis of the evolution of pathway knowledge and annotations showed that \textbf{\color{oxygenpurple}the use of outdated resources has strongly affected practical genomic analysis and recent literature.} 
\vfill
$67\%$ of $3,900$ publications we surveyed in 2015 referenced outdated software that captured only $26\%$ of biological processes and path-ways identified using current resources.
\vfill
The very popular DAVID tool, revised in January 2010! 
\vfill
\textbf{\color{oxygenrose}Remarkably, a total of 2,601 publications from 2015 $(67\%)$ cited severely outdated tools!}
\vfill
At least semiannual updates should be required, as major databases release several versions annually. To ensure reproducibil-ity, tools need to provide historical gene annotations. As an example of recommended practice, fry shiny tool is synchronized quarterly with the Ensembl database and maintains archived versions dating to 2011. Reliable up-to-date software allows researchers to make the best use of current knowl-edge of gene function and interrogate experimental data for scientific  discoveries.
\vfill 
\hfill
\footnotesize Wadi \emph{et al.}, Nature(2016)
\end{frame}

\begin{frame}[plain,t]
	\vspace{0.7cm}
	\textbf{\huge{Acknowledgements}}\\
	\vspace{1cm}
	
	\begin{minipage}[t]{0.49\textwidth}
	{\color{oxygenpurple}\textbf{WEHI Bioinformatics}}\\
	\vspace{0.20cm}
	
	\textbf{Gordon K. Smyth}\\
	Guido Pacini\\
	\vspace{0.35cm}
	
	{\color{oxygenpurple}\textbf{WEHI Population Health \\and Immunity}}\\
	\vspace{0.20cm}
	
	Saskia Freytag\\
	Roberto Bonelli\\
	\vspace{0.7cm}
	\end{minipage}%
	\begin{minipage}[t]{0.49\textwidth}
	{\color{oxygenpurple}\textbf{Data}}\\	
	{\color{oxygenrose}\textbf{Breast Cancer Laboratory Stem Cells and Cancer Division}}\\
	\vspace{0.20cm}
	
	Jane Visvader\\
	Delphine Merino\\
	Bhupinder Pal
	\vspace{0.75cm}
	
	\includegraphics[width=1\textwidth]{WEHI}
	\vspace{1.35cm}
	
	
	\end{minipage}
\end{frame}

%\begin{frame}
%\url{http://shiny.bioinf.wehi.edu.au/giner.g/FRY_GeneSetExplorerApp/}{Fry_Shiny_App}
%\end{frame}
\end{document}

